app/views/layouts/_base.html.erb

```diff
@@ -186,7 +186,7 @@
 
   </script>
   <%# END MAS JAVASCRIPT SETUP %>
-
+  <meta name="p:domain_verify" content="96b592752f8004b8838349b27c7355ba"/>
```

Comment:
does this have to go at the bottom of the `<head>` or can we put it at the top with all the other meta tags?

Reply:
I think I would prefer to see this with the other tags as well. I don't believe there would be any effect on performance. Not critical though since this addition is temporary.

Reply:
@willhall88 @davidtrussler 👍 moved